prompts/export_ansible_execution_task.md

```diff
@@ -0,0 +1,43 @@
+Process ALL migration checklist items for module: {module}
+
+CHEF SOURCE PATH: {chef_path}
+ANSIBLE OUTPUT PATH: {ansible_path}
+
+MIGRATION PLAN FOR REFERENCE:
+{migration_plan}
+
+CHECKLIST TO PROCESS:
+{checklist}
+
+CRITICAL INSTRUCTIONS:
+You MUST process EVERY SINGLE ITEM in the checklist that is marked as "pending", "missing", or "error".
+
+Your task:
+1. Go through the ENTIRE checklist from top to bottom
+2. For EACH item that needs processing:
+   a. Read the Chef source file (if it exists, skip if "N/A")
+   b. Convert it to the appropriate Ansible format
+   c. Write it to the target Ansible location using the correct tool
+   d. Move on to the next item
+3. Do NOT stop after creating just one or two files
```

Comment:
`Do NOT stop after creating just one or a few files, repeat the loop for all checklist items`